sources/PTYSession.h

```diff
@@ -1,28 +1,23 @@
 // Implements the model class for a terminal session.
 
-#import "Api.pbobjc.h"
```

Comment:
This is used in this file. For example, `ITMGetBufferResponse`. Anything with the `ITM` prefix comes from the protobufs. However, since only objects are ever referenced this import could be replaced with forward references like `@class ITMGetBufferResponse` for each referenced symbol.